app/models/report.rb

```diff
@@ -21,33 +21,33 @@ def created_on
     created_at.to_date
   end
 
-  def mentioning_report_ids
+  def find_report_ids
```

Comment:
長くなるので好き嫌いが分かれるかもしれませんが、`_in_content`を足して上げた方がより明示的な名前になりそうな気がします。
（`find_report_ids`だと「レポートがレポートのidを見つけるってどういうこと？」って思われそう）

```suggestion
  def find_report_ids_in_content
```